tests-js/Utils/initializer.js

```diff
@@ -77,6 +77,15 @@ const getPriceFeeds = async (WRBTC, SUSD, RBTC, sovryn, BZRX) => {
 	return feeds;
 };
 
+const getPriceFeedsRBTC = async (WRBTC, SUSD, RBTC, sovryn, BZRX) => {
+	const feeds = await PriceFeedsLocal.new(WRBTC.address, BZRX.address);
+
+	await feeds.setRates(WRBTC.address, RBTC.address, oneEth.toString());
+	await feeds.setRates(WRBTC.address, SUSD.address, oneEth.toString());
```

Comment:
looks like py code for this:
`feeds.setRates(
            RBTC.address,
            SUSD.address,
            1e22
        )`
```suggestion
	await feeds.setRates(WRBTC.address, SUSD.address, new BN(10).pow(new BN(22)).toString());
```
or pls provide the origin source